avm/utilities/pipelines/publish/Confirm-ModuleIsPublished.ps1

```diff
@@ -0,0 +1,81 @@
+<#
+.SYNOPSIS
+Check if a module in a given path is published in a given version
+
+.DESCRIPTION
+Check if a module in a given path is published in a given version. Tries to find the module for a maximum of 6 minutes.
```

Comment:
So should we search for 6 minutes or 1 hour?

Reply:
Both. The one 1h is contained in the 6 minutes.